CONTRIBUTING.md

```diff
@@ -1,54 +1,59 @@
 # CONTRIBUTING
-
-Contributing to the official buyTogether Application
+Contributing to the official buyTogether application
 Welcome! We're so excited that you're considering contributing to the official GitHub page for buyTogether.
-
 ---
-
-## Make changes locally
-
-- Fork the repo so that you can make your changes without affecting the original project until you're ready to merge them.
-
-- Create a working branch 
-
-- Set up a development environment to begin working on the project all you need to do is 'vagrant up'
-
+## Prerequisites
+- Fork the repo to make your changes without affecting the original project until you're ready to merge.
+- Create a working branch.
+- Set up a development environment to begin working on the project, all you need to do is 'vagrant up'
 - Start with your changes!
-
 ---
-
-## Commit your update
-Commit the changes once you are happy with them. 
-
+## Commiting
+Please use the guide below inorder to follow our commit message convention:
+* [commit convention](https://cbea.ms/git-commit/#seven-rules)
 ---
-
 ## Pull Request
 - When you're finished with the changes, create a pull request, also known as a PR.
-- Please note that at least two team members of the group should review and approve the pull request before asking the mentors to review and merge.
-- The reviewer resolve the conversation to indicate that they are ok with the updated content.
+- If your PR depends on the merging of other PRs, indicate this on the commit message or on the comments.
+Specifically, if it contains unmerged commits of any other PRs,
+indicate to the reviewers which commits are expected to be reviewed as part of your PR.
+- Keep your PRs as small as possible.
+Ideally, a single PR should include a single commit.
+- The reviewer resolves the conversation to indicate that they are ok with the updated content.
 - If you run into any merge issues, check how to solve them and resolve merge conflicts and other issues.
-
+- Changes made as part of the review process (i.e. fixes to the content of the original commit) ,
+should be added to the original commit (e.g. using `commit --amend`), rather than creating a new commit. 
+- Please keep in mind that at least two team members of the group should "quality review" (i.e. NO LGTM)
```

Comment:
`should provide a sufficient feedback` seems better IMO. there is a time and a place for LGTM, but it can't be the case in most of the time.